src/panels/config/energy/dialogs/dialog-energy-solar-settings.ts

```diff
@@ -130,6 +130,7 @@ export class DialogEnergySolarSettings
                     style="display: flex; align-items: center;"
                   >
                     <img
+                      alt=""
```

Comment:
There text accompanying this image, therefore I believe the image to be decorative.